examples/minimal/packages/client-react-external-wallet/src/mud/systemCalls.ts

```diff
@@ -1,15 +1,53 @@
-import { type WalletClient } from "viem";
-import { type MUDNetwork } from "./NetworkContext";
+import { parseEther, type Hex } from "viem";
+import { resourceToHex } from "@latticexyz/common";
+import { encodeSystemCallFrom } from "@latticexyz/world";
+import { type MUDNetwork, type ExternalWalletClient } from "./NetworkContext";
 import IWorldAbi from "contracts/out/IWorld.sol/IWorld.abi.json";
 
-export const increment = async (client: WalletClient, network: MUDNetwork) => {
+// Executes the `increment` system call on behalf of the external wallet using the burner wallet.
+// This is solely for demonstrating burner wallet delegation, as the `increment` call can actually be made by any sender.
+export const increment = async (externalWalletClient: ExternalWalletClient, network: MUDNetwork) => {
   const { request } = await network.publicClient.simulateContract({
+    account: network.burnerWalletClient.account,
+    blockTag: "pending", // burner uses this tag
```

Comment:
I've realized that manually specifying blockTag here is cumbersome. I'll move this to the burner actions.

Reply:
This has been fixed.